src/olympia/lib/crypto/tests/test_signing.py

```diff
@@ -602,21 +602,7 @@ def test_sign_bump_non_ascii_filename(self, mock_sign_file):
         tasks.sign_addons([self.addon.pk])
         assert mock_sign_file.called
         self.version.reload()
-        assert self.version.version == '0.0.1.1-signed'
-        self.file_.reload()  # Otherwise self.file_.file doesn't get re-opened
-        assert file_hash != self.file_.generate_hash()
-        self.assert_backup()
-
-    @mock.patch('olympia.lib.crypto.tasks.sign_file')
-    def test_sign_bump_non_ascii_version(self, mock_sign_file):
-        """Sign versions which have non-ascii version numbers."""
```

Comment:
We don't have such versions in the database. Version numbers have to be ascii by definition, this is enforced by both [addons-server](https://github.com/mozilla/addons-server/blob/9585c60ec916ad9db58d9c9943ed042123c48e7c/src/olympia/files/utils.py#L46) and addons-linter.